spec/intro.dd

```diff
@@ -12,18 +12,18 @@ other documents$(COMMA) see $(LINK2 https://dlang.org/, The D Language Website).
 
 $(H2 Phases of Compilation)
 
-$(P The process of compiling is divided into multiple phases. Each phase has no
-dependence on subsequent phases. For example, the scanner is not perturbed by
-the semantic analyzer. This separation of the passes makes language tools like
-syntax directed editors relatively easy to produce. It also is possible to
+$(P The process of compiling is divided into multiple phases. Each phase is
+independent of subsequent phases. For example, the scanner is not affected by
+the semantic analyzer. This separation of passes makes language tools like
+syntax directed editors relatively easy to create. It is also possible to
 compress D source by storing it in $(SINGLEQUOTE tokenized) form.)
 
 $(OL
         $(LI $(B source character set)$(BR)
 
-        The source file is checked to see what character set it is,
+        The source file is checked to see which encoding it is using,
         and the appropriate scanner is loaded. ASCII and UTF
-        formats are accepted.
```

Comment:
This whole paragraph is a bit superfluous TBH. ASCII is a subset of UTF-8 after all. So something as simple as "Any UTF-encoded source file is valid." might be enough, wouldn't it ?